README.md

```diff
@@ -250,7 +250,7 @@ Props:
 ### Users slice
 Location: `src/modules/users/users-slice.js`
 
-The same actions and reducer, but created with `Redux Toolkit`.
+The same action and reducer, but created with `Redux Toolkit`.
```

Comment:
why singular?

Reply:
Because only one action called `ADD_USER` exists at this point